MapleServer2/Servers/Game/FieldManager.cs

```diff
@@ -25,6 +25,8 @@ public class FieldManager
         public readonly FieldState State = new FieldState();
         private readonly HashSet<GameSession> Sessions = new HashSet<GameSession>();
 
+        private Task HealingSpot;
```

Comment:
The word `Task` or `Thread` should be added to the name of this variable because it could be confused with an actual healing spot, so maybe `HealingTask` or `HealingSpotTask`

Reply:
Agree